sympy/series/tests/test_limitseq.py

```diff
@@ -83,6 +83,9 @@ def test_limit_seq():
 
     raises(ValueError, lambda: limit_seq(e * m))
 
+    # issue 19868
+    assert limit_seq(1/gamma(n+S.One/2), n) == 0
```

Comment:
Should be like this:
```
def test_issue_19868():
    assert limit_seq(1/gamma(n + S.One/2), n) == 0
```

Reply:
Also maintain two line spacing between the issues.

Reply:
Gotcha, thanks